### DISCUSSION TITLE
Has classNames been removed?

### DISCUSSION THREAD

**Discussion by baufometic** [react-icons/react-icons#758] [Q&A]
There is a chance that something in my repo has changed, but up until recently I was able to pass the prop 'className' directly to the icon as such, and it would apply styles:

```javascript
import { BiUser } from "react-icons/bi";

return(
  <BiUser className="icon" />
)
```

Has something changed in recent updates, or is it just my linter? Now I get the following error:

Type '{ className: string; }' is not assignable to type 'IntrinsicAttributes & IconBaseProps'.
  Property 'className' does not exist on type 'IntrinsicAttributes & IconBaseProps'.

**Comment by TimotiusAnrez** [comment]
I also has the same issue, not sure why is this happening tho...

**Comment by 1Medo** [comment]
If you're facing an issue where **className** is not being recognized and you're getting a type error, the solution is simple.

In the **iconBase.d.ts** file located in **node_modules/react-icons/lib/**
you need to add the **className** property to the **IconBaseProps** interface to avoid the type error.

Add this line to the interface:
```
export interface IconBaseProps extends React.SVGAttributes<SVGElement> {
    children?: React.ReactNode;
    size?: string | number;
    color?: string;
    title?: string;
    className?: string; // Add this line
}
```
Once you add this line, you should be able to use **className** without any issues.
